### PR TITLE
DDP-4709: Consent styling

### PR DESCRIPTION
**Pull request by fat23cat**
## Context

Added CSS classes for better styling and new consent header.

### CODE REVIEW

File: study-builder/studies/testboston/i18n/en.conf [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
   "consent": {
     "name": "Consent Form",
-    "title": "Consent Form",
-    "subtitle": "Download Form (Read-Only)",
+    "title": "",
+    "subtitle": "<div class="consent-header"><h1 class="consent-header__title">Consent Process</h1><a class="button button_download" href="consent.pdf" download><img src="assets/images/download.svg" alt="Download file" class="consent-header__icon">Download this document</a></div>",
```
Comment (fat23cat) [inline study-builder/studies/testboston/i18n/en.conf]: I'm not confident that's correct, how can we add HTML code here?

Comment (yufengwng) [reply study-builder/studies/testboston/i18n/en.conf]: Looks fine. The `subtitle` stuff doesn't use templates, just a `text`. So putting the HTML here is the way I will have done it too.

Comment (fat23cat) [reply study-builder/studies/testboston/i18n/en.conf]: Thanks!

Comment (yufengwng) [reply study-builder/studies/testboston/i18n/en.conf]: Will it be easier to put this in `title` instead? We can make changes on server-side to support HTML in title.

Comment (fat23cat) [reply study-builder/studies/testboston/i18n/en.conf]: If it doesn’t take much time, then let's do it.
I'll update my HTML code.

Comment (yufengwng) [reply study-builder/studies/testboston/i18n/en.conf]: Here it is: https://github.com/broadinstitute/ddp-study-server/pull/256

Comment (fat23cat) [reply study-builder/studies/testboston/i18n/en.conf]: Thanks! I also updated my PR.
If we merge our PRs today could you please re-create testboston consent activity? I want to make sure tomorrow that everything is good.

**Review by yufengwng — APPROVED** [review]
Looks good. Let's get this merged.

**Pull request merged**